docs/articles/actors/receive-actor-api.md

```diff
@@ -843,6 +843,38 @@ The result of this is that when an actor is restarted, any stashed messages will
 > [!NOTE]
 > If you want to enforce that your actor can only work with an unbounded stash, then you should use the `IWithUnboundedStash` interface instead.
 
+### Bounded Stashes
+
+In certain scenarios, it might be helpful to put a limit on the size of the `IStash` inside your actor. You can configure a bounded stash via the following actor definition:
```

Comment:
Documented how to configure a bounded stash with both `Props` and `akka.actor.deployment` here.